.gitignore

```diff
@@ -2,3 +2,4 @@
 .gitignore
 yarn-error.log
 node_modules/
+.idea
```

Comment:
please remove this file from this commit; the `.idea` directory is already in the file `.gitignore`